### PR TITLE
add more css

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

File: manifest.json.tmpl [hunk 1]
```diff
@@ -22,6 +22,8 @@
       "default_panel": "sidebar.html"
     },
     "web_accessible_resources": [
+      "images/globe.svg",
```
Comment (johngruen) [inline manifest.json.tmpl]: Actually don't need these...

**Comment by pdehaan** [comment]
Current [**stylelint**](http://npm.im/stylelint) output:

```sh
$ $(npm bin)/stylelint 'addon/**/*.css'

addon/sidebar.css
  67:18  ✖  Expected "#ffffff" to be "#fff"                                                            color-hex-length
  74:11  ✖  Unexpected missing generic font family                                                     font-family-no-missing-generic-family-keyword
 135:5   ✖  Unexpected shorthand "background" after "background-size"                                  declaration-block-no-shorthand-property-overrides
 161:16  ✖  Expected whitespace before "*/"                                                            comment-whitespace-inside
 217:16  ✖  Expected "translate3D" to be "translate3d"                                                 function-name-case
 246:14  ✖  Unexpected unit                                                                            length-zero-no-unit
 291:1   ✖  Expected selector ".toolbar__home" to come before selector ".toolbar__home:hover"          no-descending-specificity
 291:1   ✖  Expected selector ".toolbar__home" to come before selector ".toolbar__home:focus"          no-descending-specificity
 291:1   ✖  Expected selector ".toolbar__home" to come before selector ".toolbar__home:active"         no-descending-specificity
 295:1   ✖  Expected selector ".toolbar__refresh" to come before selector ".toolbar__refresh:hover"    no-descending-specificity
 295:1   ✖  Expected selector ".toolbar__refresh" to come before selector ".toolbar__refresh:active"   no-descending-specificity
 307:1   ✖  Unexpected missing end-of-source newline                                                   no-missing-end-of-source-newline


$ cat .stylelintrc
{
  "extends": "stylelint-config-standard",
  "rules": {
    "indentation": null
  }
}
```

I can also comment on these in-line if you prefer.
I also had to disable the `indentation` rule since we're using a 4 space indent and it seems to expect 2.

**Comment by johngruen** [comment]
Okay i actually added stylelint to the project

**Comment by johngruen** [comment]
@ianb i think you might want to rebase some of your smaller PRs against this since it pretty much takes care of a good chunk of the styles/interactions we'll need

**Pull request merged**